src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogComputerPublisher.java

```diff
@@ -80,33 +79,43 @@ protected void doRun() throws Exception {
             // Add JenkinsUrl Tag
             globalTags = TagsUtil.addTagToTags(globalTags, "jenkins_url", DatadogUtilities.getJenkinsUrl());
             for (Computer computer : computers) {
+                Map<String, Set<String>> tags = TagsUtil.merge(
+                        DatadogUtilities.getComputerTags(computer), globalTags);
```

Comment:
Yeah, true. I can look into it. Honestly, I'm not sure if the hostname tags are needed/helpful for these metrics so it might be worth using a new method that doesn't calculate hostname.

Reply:
One thing to note is I'm not changing the amount of times `getComputerTags` is called, this line is just moving it up in the loop: https://github.com/jenkinsci/datadog-plugin/pull/71/files#diff-e231a7baaf7bf9e40bbcfcd53c65b47cL95

Reply:
Oh that's correct, thanks I missed that 👍

Reply:
The method is pretty time consuming, though (locally running my new tests it takes 22.465s). It's definitely something to keep in mind when thinking of improvements to clients.